mkinitcpio

```diff
@@ -408,8 +410,8 @@ while :; do
             ;;
         -D|--hookdir)
             shift
-            _d_hooks="$(readlink -f "$1")/hooks"
-            _d_install="$(readlink -f "$1")/install"
+            _d_flag_hooks+="$1/hooks:"
+            _d_flag_install+="$1/install:"
```

Comment:
You either need to build this as an array, or trim the trailing ":" when you reassign to _d_hooks, et al at the end. An empty element in the PATH refers to $PWD, which isn't something we want.

Reply:
Fixed